core/src/main/java/com/yahoo/ycsb/Client.java

```diff
@@ -553,6 +668,20 @@ private static void exportMeasurements(Properties props, int opcount, long runti
       exporter.write("OVERALL", "RunTime(ms)", runtime);
       double throughput = 1000.0 * (opcount) / (runtime);
       exporter.write("OVERALL", "Throughput(ops/sec)", throughput);
+      
+      boolean trackJVMStats = props.getProperty(Measurements.MEASUREMENT_TRACK_JVM_PROPERTY, 
```

Comment:
OK, I wasn't sure if ya'll were ok with adding stats like that overall but it would be great to have all the time.